Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.yml

```
@@ -5827,6 +5827,45 @@ script:
     deprecated: false
     arguments: []
     outputs: []
+  - arguments:
+    - default: false
+      description: The remote incident or detection ID.
```

```suggestion
      description: The remote ticket ID.
```
Is it called a ticket or an incident? Ensure that everything has the same name.

---

changed

---

done